src/walletConnect/request.test.ts

```diff
@@ -154,28 +138,33 @@ describe(handleRequest, () => {
       .run()
   })
 
-  it('supports personal_sign', async () => {
+  it('supports personal_sign, including for an unsupported chain', async () => {
     await expectSaga(handleRequest, personalSignRequest)
       .withState(state)
       .call([viemWallet, 'signMessage'], { message: { raw: 'Some message' } })
       .run()
+
+    await expectSaga(handleRequest, { ...personalSignRequest, chainId: 'eip155:unsupported' })
+      .withState(state)
+      .call([viemWallet, 'signMessage'], { message: { raw: 'Some message' } })
+      .run()
   })
 
-  it('supports eth_signTypedData', async () => {
+  it('supports eth_signTypedData for supported chain', async () => {
```

Comment:
nit: i'm a bit confused with teh logic of adding the "supported chain" mention.

i'm doubtful that it's worth to mentioning "supported chain" here, since we don't have the opposite test (throws for unsupported chain).

however, for `eth_signTransaction` and `eth_sendTransaction` we have both. i would expect to have "supported"/"unsupported" in their naming, rather than here.